src/about.html

```diff
@@ -150,3 +153,27 @@ <h3 class="text-dark">Ready to work together?</h3>
         </div>
     </div>
 </section>
+
+<script>
+    const randomizeTeam = function() {
+        // get an array of all team members
+        let members = [...document.querySelectorAll(".team-member")];
+        let currentIndex = members.length;
+        // and shuffle
+        while (currentIndex != 0) {
+            let shuffleIndex = Math.floor(Math.random() * currentIndex);
+            currentIndex--;
+            [members[currentIndex], members[shuffleIndex]] = [members[shuffleIndex], members[currentIndex]];
+        }
+        // then override the flex order with new shuffled index
+        // hiding all but the first 5
+        members.forEach((m, i) => {
+            m.style.order = i;
```

Comment:
I did not know about flex [`order`](https://developer.mozilla.org/en-US/docs/Web/CSS/order) - that's cool!